app/src/main/java/com/willowtree/vocable/CategoriesUseCase.kt

```diff
@@ -49,10 +46,9 @@ class CategoriesUseCase(
     }
 
     override suspend fun addCategory(categoryName: String, sortOrder: Int) {
-        presetsRepository.addCategory(
```

Comment:
Is the `PresetsRepository.addCategory` still used?

Reply:
Great ask, looks like it's not